content/en/tracing/trace_collection/compatibility/ruby.md

```diff
@@ -1,56 +1,56 @@
 ---
```

Comment:
Do we still need this file, now that it is single sourced?

Reply:
Nope! Removed it. Thanks!